app/assets/stylesheets/components/common/_social-sharing.scss

```diff
@@ -0,0 +1,34 @@
+.list {
+  > li.social-sharing__item:before {
```

Comment:
Can we move this style so it sits within the `.social-sharing` namespace? Strictly speaking, components shouldn't really be modifying other components.